index.php

```diff
@@ -132,15 +140,47 @@
             $url = sprintf('team/%s', $platform);
             break;
     }
+    return $verifyUrl($url) ? $routes[$url] : $routes['default'];
+};
+
+$langUrlBuilder = function ($url, $platform) use ($verifyUrl, $routes) {
+    if ($verifyUrl($url)) {
+        return $routes[$url] . "/" . $platform;
+    } else {
+        return $routes['default'];
+    }
+};
+
+$langs = ['ar', 'bg', 'bs', 'cs', 'de', 'el', 'en', 'es', 'fr', 'hr', 'hu', 'id', 'ja', 'pl', 'pt', 'ro', 'ru', 'tr', 'uk', 'vi', 'zh'];
 
```

Comment:
We need to devise a way to update this array of languages when we add a new language into crowdin

Reply:
Hard coded for now. We need to do the same thing to that one :/ The website is a bit more tricky since there are also flags that need to be shown for the dropdown menu.

Reply:
Updated this with automated deploy to build this include file.